apis/Google.Cloud.Firestore/docs/userguide.md

```diff
@@ -99,21 +94,6 @@ In this section, we'll deal with a simple document that just has a single counte
 and periodically (once per day, for example) we'll update another counter to match the current value. For more
 details of counters, see the [main Firestore user guide](https://firebase.google.com/docs/firestore/solutions/counters).
 
-### Creating multiple documents atomically
-
-If we only have a single current counter and a single daily counter, we could 
-
-[!code-cs[](obj/snippets/Google.Cloud.Firestore.UserGuide.txt#TransactionSyncCallbackNoResult)]
```

Comment:
Should this snippet (and below) also be removed?

Reply:
Yes - that's actually the warning I'm fixing :) Those snippets are gone (#1944) but I hadn't updated the docs. There are probably other doc updates I've missed, and I'll take another pass at all the docs before the next release, but for now I just wanted to remove the warnings from the build.